.readthedocs.yaml

```diff
@@ -5,6 +5,8 @@ build:
   tools:
     python: "mambaforge-4.10"
   jobs:
+    post_checkout:
+      - git fetch --unshallow  || true
```

Comment:
Maybe add a comment pointing to the issue that causes us not to be able to use shallow clones? It  does seem weird that the whole git history would be needed. Or is this perhaps to point to changelogs of previous versions or so?

Reply:
I actually don't know if this is actually needed. I am just responding to the notice I got from RTD. Now that I look in the RTD admin settings, I cannot even find where they would have detected this to trigger the memo to begin with.

@astrofrog or @saimn , do you remember if we need this or can we simply ignore the RTD memo?

Reply:
We need the commits up to the latest tag, so that setuptools_scm can compute a version number. 
So we don't need all the commits, and maybe we can use instead `--shallow-since=<date>` where data would be just before the last release (or maybe the base commit of the v5.0.x since we may want to backport this).

Reply:
@saimn , base commit means the hash at 5.0 tag time or just whatever from the branch? Do you just want to post a code suggestion with it or push here directly? 🙏